src/Type/ObjectType/RootQuery.php

```diff
@@ -610,7 +610,7 @@ function ( $post ) use ( $allowed_post_types ) {
 						'type'        => 'User',
 						'description' => __( 'Returns the current user', 'wp-graphql' ),
 						'resolve'     => function ( $source, array $args, AppContext $context ) {
-							return isset( $context->viewer->ID ) && ! empty( $context->viewer->ID ) ? DataSource::resolve_user( $context->viewer->ID, $context ) : null;
+							return ! empty( $context->viewer->ID ) ? DataSource::resolve_user( $context->viewer->ID, $context ) : null;
```

Comment:
I don't care _that much_ but I've been getting rid of more and more of the references to DataSource methods and calling the loaders directly, like so: 

```suggestion
							return ! empty( $context->viewer->ID ) ? $context->get_loader( 'user' )->load_deferred( $context->viewer->ID ) : null;
```